app/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
         versionCode 1
         versionName '1.0.0'
 
-        minSdkVersion 17
+        minSdkVersion 14
```

Comment:
minSdkVersion 17 is the lowest version that our non-support setup in **master** supports due to `Fragment.onViewStateRestored` only being available starting at API level 17.